src/pet.js

```diff
@@ -0,0 +1,84 @@
+const AGE_INCREMENT = 1; 
+const MAXIMUM_AGE = 30;
+
+const HUNGER_INCREMENT = 5;
+const HUNGER_DECREMENT = 3;
+const MINIMUM_HUNGER = 0;
+const HUNGER_THRESHOLD = 5;
+const MAXIMUM_HUNGER = 10;
+
+const FITNESS_DECREMENT = 3;
+const FITNESS_INCREMENT = 4;
+const MAXIMUM_FITNESS = 10;
+const FITNESS_THRESHOLD = 3;
+const MINIMUM_FITNESS = 0;
+
+const DEFAULT_CONDITION = "I feel great!";
+const DEATH_STATEMENT = 'Your pet is no longer alive :(';
+
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = MINIMUM_HUNGER;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = [];
+} 
+Pet.prototype = {
+    get isAlive() {
+      return this.age < MAXIMUM_AGE && this.hunger < MAXIMUM_HUNGER && this.fitness > MINIMUM_FITNESS;
+    }
+  };
+Pet.prototype.growUp = function() {
```

Comment:
In order to avoid constantly repeating `Pet.prototype.functionName = function() {...};` you can create a single Pet.prototype object, which will have all the methods inside, as `methodName() {...}`.